go/libkb/bootstrap_active_device.go

```diff
@@ -78,21 +78,23 @@ func bootstrapActiveDeviceReturnRawError(m MetaContext, uid keybase1.UID, device
 		m.Debug("active device is current")
 		return nil
 	}
-	uv, sib, sub, deviceName, err := LoadUnlockedDeviceKeys(m, uid, deviceID, online)
+	uv, sib, sub, deviceName, err := LoadUnlockedDeviceKeys(m, uid, deviceID, online, false)
 	if err != nil {
 		return err
 	}
 	err = m.SetActiveDevice(uv, deviceID, sib, sub, deviceName, KeychainModeOS)
 	return err
 }
 
-func LoadUnlockedDeviceKeys(m MetaContext, uid keybase1.UID, deviceID keybase1.DeviceID, online bool) (uv keybase1.UserVersion, sib GenericKey, sub GenericKey, deviceName string, err error) {
+func LoadUnlockedDeviceKeys(m MetaContext, uid keybase1.UID, deviceID keybase1.DeviceID, online bool, staleOK bool) (uv keybase1.UserVersion, sib GenericKey, sub GenericKey, deviceName string, err error) {
```

Comment:
Sorry to send this back, but as a stylistic thing, I only approve of one bool per argument list. Can you make a `LoadUnlockedDeviceKeyOpts` object so that we don't mix these up? Or an enum? Do we allow all 4 possibilities?